aptos-move/e2e-move-tests/src/tests/scripts.rs

```diff
@@ -80,3 +81,74 @@ fn read_coin(h: &MoveHarness, account: &AccountAddress) -> u64 {
         .unwrap()
         .coin()
 }
+
+#[test]
+fn test_two_to_two_transfer_gas_payer() {
```

Comment:
Can you add some more tests around cases such as:
1. Feature flag is not enabled.
2. Gas payer doesn't have enough gas
3. Extra senders without gas payer bit set.
4. etc. Any other edge cases? Let's be as thorough as possible here to make sure there are no unforeseen issues

Reply:
Still missing some failure cases:
1. Gas payer doesn't have sufficient balances.
2. Invalid gas payer signature
3. Etc.
I'll leave it to you to determine what can be covered with unit vs move e2e tests. We'll also have a more concise integration test later in TS as well for a true e2e.

Reply:
Given that this code is just using multi-agent signing and reusing the epilogue. Most of these flows are tested by regular tests

Reply:
I have tests that cover (scripts/entry functions)/(no signer/single signer/multiple signers)/(gas bit/no-gas bit)(feature/no feature)